spec/board_spec.rb

```diff
@@ -14,7 +14,7 @@
     end
 
     it 'defaults to a 4x4 board upon creation' do
-      @board.cells
+      @board.board_cells
```

Comment:
Let's mention this to our beloved instructors since it doesn't follow interaction pattern.
Due to our optional dynamic method for creating a board, it overlapped with the stored attributes in initialize.